backend/wmg/data/cube_pipeline.py

```diff
@@ -28,9 +30,10 @@ def load_data_and_create_cube(path_to_datasets: str, corpus_name: str, log_level
     A per tissue mapping of cell ontologies is generated and the files are copied to s3 under a shared timestamp,.
     On success the least recent set of files are removed from s3
     """
-    copy_datasets_to_instance(path_to_datasets)
+    s3_uris = get_s3_uris()
+    copy_datasets_to_instance(s3_uris, path_to_datasets)
     load(path_to_datasets, corpus_name)
-    create_cube(corpus_name)
+    create_cube(corpus_name)  # Todo dry up with create cube func in fixtures
     cell_type_by_tissue = get_cells_by_tissue_type(corpus_name)
     generate_cell_ordering(cell_type_by_tissue)
     update_s3_resources()
```

Comment:
beyond the scope of this PR, but shouldn't we log progress?  output counts of datasets processed? artifacts output?

Reply:
Yea -- I think there notes about that in the productionazation ticket https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1926